### DISCUSSION TITLE
Possible to have an OLED display and menu?

### DISCUSSION THREAD

**Discussion by MalvareRJ** [erichelgeson/BlueSCSI-v1#211] [Ideas]
Is it possible to implement a simple menu to locally mount images?
This would be very handy specially for CD images.
I have really no clue if there's room and processing power left for this. So my appologies if I sound stupid!

Thanks!

**Comment by erichelgeson** [comment]
Currently we really don't have enough pins free for this, though would be a cool idea. Depending on the platform we will have host apps to allow CD switching, macintosh first, but eventually others.
